Tools/autotest/arducopter.py

```diff
@@ -125,6 +125,12 @@ def close(self):
         if self.copy_tlog:
             shutil.copy(self.logfile, self.buildlog)
 
+    def get_rudder_channel(self):
+        return int(self.get_parameter("RCMAP_YAW"))
+
+    def get_disarm_delay(self):
+        return self.get_parameter("DISARM_DELAY")
```

Comment:
Technically this is an int as well.  Casting the results in the previous  getters is a bit gratuitous, 'though.